.github/workflows/Documenter.yml

```diff
@@ -0,0 +1,18 @@
+name: Documenter
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+
+jobs:
+  Documenter:
+    name: Documentation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/julia-buildpkg@latest
+      - uses: julia-actions/julia-docdeploy@latest
+        env:
+          GITHUB_TOKEN: \${{ secrets.GITHUB_TOKEN }}
+          DOCUMENTER_KEY: \${{ secrets.DOCUMENTER_KEY }}
```

Comment:
Not sure the backslashes are correct here

Reply:
Thanks!